spec/jobs/scrape_keyword_job_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ScrapeKeywordJob do
+  include ActiveJob::TestHelper
+
+  describe '#perform' do
+    context 'given an ERROR occurs' do
+      context "when the keyword doesn't exist" do
+        it 'DOES NOT retry' do
+          perform_enqueued_jobs do
+            described_class.perform_later(keyword_id: 42)
+          end
+
+          assert_performed_jobs 1
```

Comment:
MAybe missing parentheses for code consistency?

```suggestion
          assert_performed_jobs(1)
```

Reply:
Fixed in 810d359d3574f9595bdc925c5eb031821d37138a

Not sure why Rubocop lets this passes